core/app/components/MemberSelect/MemberSelectAddUserButton.tsx

```diff
@@ -9,27 +9,33 @@ import { MemberSelectAddUserForm } from "./MemberSelectAddUserForm";
 type Props = {
 	community: Communities;
 	email: string;
+	onUserAdded: () => void;
 };
 
 export const MemberSelectAddUserButton = (props: Props) => {
 	const [open, setOpen] = useState(false);
 
-	const onClick = useCallback((e: React.MouseEvent<HTMLButtonElement>) => {
-		e.preventDefault();
```

Comment:
I don't think we actually need these `e.preventDefault()`s if we specify the button type to be 'button'